docs/src/modules/java/partials/query-syntax-reference.adoc

```diff
@@ -166,6 +169,60 @@ In some use cases it is important to explicitly express that a value is missing,
 
 Optional fields with values present can be queried just like regular view fields:
 
+// end::grpc[]
+
+
+// tag::spring[]
+
+When modeling your queries, the following data types are supported:
+
+[cols="1,1"]
+|===
+| Data type | Java type
+
+| Text
+| `String`
+
+| Integer
+| `int` / `Integer`
+
+| Long (Big Integer)
+| `long` / `Long`
+
+| Float (Real)
+| `float` / `Float`
+
+| Double
+| `double` / `Double`
+
+| Boolean
+| `boolean` / `Boolean`
+
+| Byte String
+| `ByteString`
+
+| Array
+| `Collection<T>` and derived
+
+// not supported at the moment, see https://github.com/lightbend/kalix-jvm-sdk/issues/1391
```

Comment:
Nice to keep a reference. Somehow unorthodox but I think it is useful.

Reply:
Yeah, I was not sure.. but seemed okay just because we have it for GRPC counter-part.